test/Microsoft.AspNetCore.Razor.Test/CodeGenerators/CSharpCodeGeneratorTest.cs

```diff
@@ -14,9 +17,12 @@ namespace Microsoft.AspNetCore.Razor.CodeGenerators
 {
     public class CSharpCodeGeneratorTest
     {
+        private static readonly string _testOutputDirectory = "TestFiles/CodeGenerator/Output/CSharpCodeGenerator";
```

Comment:
But do we want the `CSharpCodeGenerator` folder here i.e. put the files / resources in the `Output` folder?